store/todos.js

```diff
@@ -57,6 +70,7 @@ export const actions = {
       // return to the location where is was dispatched after being resolved
       setTimeout(() => {
         commit('deleteTask', val);
+        commit('setListPerPage');
```

Comment:
When we mark a task as done from the search result, all the tasks are shown instead of the searched results. this maybe happening because of setting task list here. please have a look.

Reply:
Resolved